tfhe/src/strings/ciphertext.rs

```diff
@@ -5,20 +5,25 @@ use crate::integer::{
     ServerKey as IntegerServerKey,
 };
 use crate::shortint::MessageModulus;
+use crate::strings::backward_compatibility::{FheAsciiCharVersions, FheStringVersions};
 use crate::strings::client_key::EncU16;
 use crate::strings::N;
 use rayon::iter::{IndexedParallelIterator, ParallelIterator};
 use rayon::slice::ParallelSlice;
+use serde::{Deserialize, Serialize};
 use std::borrow::Borrow;
+use tfhe_versionable::Versionize;
 
 /// Represents a encrypted ASCII character.
-#[derive(Clone)]
+#[derive(Clone, Serialize, Deserialize, Versionize)]
+#[versionize(FheAsciiCharVersions)]
 pub struct FheAsciiChar {
     pub enc_char: RadixCiphertext,
 }
 
 /// Represents a encrypted string made up of [`FheAsciiChar`]s.
-#[derive(Clone)]
+#[derive(Clone, Serialize, Deserialize, Versionize)]
+#[versionize(FheStringVersions)]
 pub struct FheString {
```

Comment:
naming mismatch between HL and this module "on purpose" ?

Reply:
Yeah, I'm not 100% attached to this name, but due to how ascii is limited, I felt like making it clear in the name might avoid some disapointments

Reply:
Fair enough